From b9e961bc481753a49685c3db1beed74aa0402468 Mon Sep 17 00:00:00 2001
From: warped-rudi <r.ihle@s-t.de>
Date: Mon, 25 Jul 2016 11:31:58 +0200
Subject: [PATCH] Remove unnecessary assign.

No need to assign queue item that will be thrown away anyway
to local variable in DiscardWorkerJobs()

Signed-off-by: warped-rudi <r.ihle@s-t.de>
---
 src/WavforHue_Thread.cpp |    5 -----
 1 file changed, 5 deletions(-)

diff --git a/src/WavforHue_Thread.cpp b/src/WavforHue_Thread.cpp
index 4ad5550..be10a20 100644
--- a/src/WavforHue_Thread.cpp
+++ b/src/WavforHue_Thread.cpp
@@ -119,15 +119,10 @@ void WavforHue_Thread::TransferQueueToThread()
 
 void WavforHue_Thread::DiscardWorkerJobs()
 {
-  SocketData putData;
-
   std::lock_guard<std::mutex> lock(gMutex);
 
   while (!gQueue.empty())
-  {
-    putData = gQueue.front();
     gQueue.pop();
-  }
 }
 //-- Threading -----------------------------------------------------
 
-- 
1.7.9.5

